src/supernode/baseclientproxy.cpp

```diff
@@ -114,48 +117,50 @@ bool supernode::BaseClientProxy::CreateAccount(const supernode::rpc_command::CRE
     std::string seed;
     wal->get_seed(seed);
     out.Seed = seed;
-    out.Result = STATUS_OK;
     return true;
 }
 
-bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out)
+bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out, epee::json_rpc::error &er)
 {
-	LOG_PRINT_L0("BaseClientProxy::GetSeed" << in.Account);
+    LOG_PRINT_L0("BaseClientProxy::GetSeed " << in.Account);
```

Comment:
Is there any sensitive info in the `in.Account` ?

Reply:
No, removed